features/discovering-rooms.feature

```diff
@@ -0,0 +1,39 @@
+@wip
```

Comment:
I think I'd prefer we use `@wip` on scenarios for the work that is actually in flight; and maybe use `@unstarted` for the scenarios that have not yet been picked up. We can use `@unscheduled` for feature definitions that we are contemplating but have not yet decided which release they go in to.

We'll probably also want to add `and not @unstarted` to the `test` script in `package.json`